### DISCUSSION TITLE
Github for Teachers: I can't enable Copilot free.

### DISCUSSION THREAD

**Discussion by maxwellamaral** [community/community#33259] [Copilot Conversations]
I'm a verified teacher on Github and I have access to some benefits on the platform. When trying to enable Copilot, in Settings -> GitHub Copilot, it asks for credit card registration for license purchase. What could be going wrong?

![image](https://user-images.githubusercontent.com/107955034/190655836-4305c23f-d9dc-4fb3-a46c-cbd0ce847d8d.png)

![image](https://user-images.githubusercontent.com/107955034/190655952-a8a4dd64-3056-4659-a0e9-3ba08924e5fc.png)

**Comment by iamcjwilliams** [comment]
Hello @maxwellamaral If you already subscribed to Copilot or signed up for the free trial, but you are interested in accessing your free Teacher subscription and you have already been verified within our GitHub Global Campus Program, you will need to cancel that subscription/trial before you can subscribe to Copilot for free. [Click through for instructions on how you can cancel your current Copilot subscription](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-subscription#canceling-your-github-copilot-subscription). Once you have successfully canceled your current subscription you can [contact Support](https://support.github.com/request) to request further instructions.

**Reply by maxwellamaral** [reply]
Thanks for the feedback @iamcjwilliams ! I am no longer registered as a trial. That's been a long time. If I register a new account as a teacher, after accepting Github, I get access to Copilot. But I would not like to use another account.
